packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -71,7 +63,11 @@ class Dropdown extends React.Component {
         }}
       >
         {toggle && Children.map(toggle, oneToggle => cloneElement(oneToggle, { parentRef: this.parentRef, isOpen }))}
-        <DropdownMenu isOpen={isOpen} aria-labelledby={id} onClick={event => onSelect && onSelect(event)}>
+        <DropdownMenu
+          isOpen={isOpen}
+          aria-labelledby={toggle ? toggle.props.id : ''}
+          onClick={event => onSelect && onSelect(event)}
```

Comment:
```suggestion
          onClick={onSelect}
```

Reply:
onSelect could be null.  I think that is why he does the check here.